shipgen/EvtCalcGenerator.h

```diff
@@ -0,0 +1,54 @@
+#ifndef EVTCALCGENERATOR_H
+#define EVTCALCGENERATOR_H 1
+
+#include "TROOT.h"
+#include "FairGenerator.h"
+#include "TTree.h"                      // for TTree
+#include "FairLogger.h"                 // for FairLogger, MESSAGE_ORIGIN
+
+class FairPrimaryGenerator;
+
+class EvtCalcGenerator : public FairGenerator
+{
+ public:
+
+  /** default constructor **/
+  EvtCalcGenerator();
+
+  /** destructor **/
+  virtual ~EvtCalcGenerator();
+
+  /** public method ReadEvent **/
+  Bool_t ReadEvent(FairPrimaryGenerator*);
+  virtual Bool_t Init(const char*, int); //!
+  virtual Bool_t Init(const char*); //!
+
+  Int_t GetNevents()
+  {
+    return fNevents;
+  }
+
+  void SetPositions(Double_t zTa=-5814.25, Double_t zDV=0.){
```

Comment:
I'd prefer not having hard-coded default arguments here.

Reply:
Commit pushed.